pubsub/mqtt3/metadata.yaml

```diff
@@ -0,0 +1,67 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: mqtt3
+version: v1
+status: stable
+title: "MQTT3"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-mqtt3/
+authenticationProfiles:
+  - title: "Connection string"
+    description: "The MQTT3 Connection/Authentication."
+    metadata:
+      - name: url
+        type: string
+        required: true
+        sensitive: true
+        description: |
+          Address of the MQTT broker.
+          Use the tcp:// URI scheme for non-TLS communication. 
+          Use the ssl:// URI scheme for TLS communication.
+        example: '"tcp://[username][:password]@host.domain[:port]"'
+metadata:
+  - name: consumerID
+    type: string
+    description: |
+      The client ID used to connect to the MQTT broker. Defaults to the Dapr app ID.
+    example: '"myMqttClientApp"'
+  - name: retain
+    type: bool
+    description: |
+      Defines whether the message is saved by the broker as the last known good value for a specified topic.
+    default: 'false'
+    example: '"true", "false"'
+  - name: cleanSession
+    type: bool
+    description: |
+      Sets the clean_session flag in the connection message to the MQTT broker if "true" (more info).
```

Comment:
"more info"?

Reply:
I think the docs had a link, which is now removed

Reply:
The `more info` link was pointing to a non-TLS [URL](http://www.steves-internet-guide.com/mqtt-clean-sessions-example/).  I have not found a reliable URL to point to. I'm open to any suggestions.

Reply:
What's the issue with the URL above?

Reply:
The URL is pointing to an unsecured location.  I believe the URLs in the Metadata yaml files should only be TLS/SSL.

Reply:
Preferably, yes. But if there's no alternative, I'm fine with a non-TLS URL. Since these links are used for docs only, a link to a non-TLS page is better than no link.